walkthroughs/eks/base.md

```diff
@@ -85,135 +84,101 @@ kubectl api-resources --api-group=appmesh.k8s.aws
-# pod/colorgateway-cc6464d75-qbznr         2/2     Running   0          1m
-# pod/colorteller-86664b5956-zhb44         2/2     Running   0          1m
-# pod/colorteller-black-6787756c7b-5sgfq   2/2     Running   0          1m
-# pod/colorteller-blue-55d6f99dc6-wltdj    2/2     Running   0          1m
-# pod/colorteller-red-578866ffb-rztfc      2/2     Running   0          1m
-#
-# NAME                        TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)    AGE
-# service/colorgateway        ClusterIP   10.100.45.69    <none>        9080/TCP   1m
-# service/colorteller         ClusterIP   10.100.186.86   <none>        9080/TCP   1m
-# service/colorteller-black   ClusterIP   10.100.52.209   <none>        9080/TCP   1m
-# service/colorteller-blue    ClusterIP   10.100.82.44    <none>        9080/TCP   1m
-# service/colorteller-red     ClusterIP   10.100.19.144   <none>        9080/TCP   1m
-#
-# NAME                                            AGE
-# virtualnode.appmesh.k8s.aws/colorgateway        1m
-# virtualnode.appmesh.k8s.aws/colorteller         1m
-# virtualnode.appmesh.k8s.aws/colorteller-black   1m
-# virtualnode.appmesh.k8s.aws/colorteller-blue    1m
-# virtualnode.appmesh.k8s.aws/colorteller-red     1m
-#
-# NAME                                                       AGE
-# virtualservice.appmesh.k8s.aws/colorgateway.appmesh-demo   1m
-# virtualservice.appmesh.k8s.aws/colorteller.appmesh-demo    1m
+NAME                                       READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.extensions/appmesh-controller   1/1     1            1           72m
+deployment.extensions/appmesh-inject       1/1     1            1           72m
+
+NAME                                      READY   STATUS    RESTARTS   AGE
+pod/appmesh-controller-5977c4b87d-68vb6   1/1     Running   1          72m
+pod/appmesh-inject-767d8c8f7c-dk4fk       1/1     Running   0          72m
+
+NAME                     TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)   AGE
+service/appmesh-inject   ClusterIP   10.100.192.220   <none>        443/TCP   72m
+```
+
+```sh
+kubectl get pod -n howto-k8s-cloudmap -o wide
+
+NAME                             READY   STATUS    RESTARTS   AGE   IP               NODE                                           NOMINATED NODE   READINESS GATES
+colorapp-blue-768d5c96b9-jdbqg   2/2     Running   0          11m   192.168.33.13    ip-192-168-56-114.us-east-2.compute.internal   <none>           <none>
+colorapp-red-9c8c4cbfc-z4k22     2/2     Running   0          11m   192.168.15.43    ip-192-168-12-143.us-east-2.compute.internal   <none>           <none>
+front-7d559c8949-qqwmb           2/2     Running   0          11m   192.168.12.218   ip-192-168-12-143.us-east-2.compute.internal   <none>           <none>
 ```
 
 Now, validate the mesh creation using the `aws` CLI:
 
 ```sh
-aws appmesh list-meshes --region us-east-2
+aws appmesh list-meshes
 # {
 #     "meshes": [
 #         {
 #             "meshName": "color-mesh",
```

Comment:
meshName needs to be updated. Similarly on L133, L138, L191

Reply:
fixed